src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/dto/sessions/CreateSessionTemplateDto.kt

```diff
@@ -58,4 +58,7 @@ data class CreateSessionTemplateDto(
 
   @Schema(description = "list of group references for allowed prisoner incentive levels", required = false)
   val incentiveLevelGroupReferences: List<String>? = listOf(),
+
+  @Schema(description = "If a session is a TAP session.", required = true)
+  val isTAPSession: Boolean,
```

Comment:
I think it should be data driven,...

Reply:
Thanks. Yes, as you suggested it is now data driven with the includeLocationGroups flag.

Reply:
Updated to be data driven as suggested with the includeLocationGroups flag.